oauth2-store/src/main/java/org/xwiki/contrib/oidc/internal/AbstractNimbusOAuth2TokenStore.java

```diff
@@ -138,7 +160,9 @@ protected NimbusOAuth2Token getToken(DocumentReference documentReference,
             if (document.isNew() || tokenObj == null) {
                 return null;
```

Comment:
You might want to also store the fact that the token does not exist (but maybe this is really just in case code for something that never really happen).

Reply:
Yes, but how to add an entry which will mean that the entry don't exist ? From what I saw in the API adding a null will remove the entry.

Reply:
Yes, the trick in this case is to insert a type which contain the value instead of the value directly.